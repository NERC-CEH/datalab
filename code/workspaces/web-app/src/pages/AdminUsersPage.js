import React from 'react';
import Page from './Page';
import AdminUsersContainer from '../containers/adminListUsers/AdminUsersContainer';

const AdminUsersPage = () => (
    <Page title="Users">
      <AdminUsersContainer />
    </Page>
);
export default AdminUsersPage;
