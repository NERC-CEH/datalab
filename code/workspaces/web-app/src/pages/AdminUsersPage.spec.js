import React from 'react';
import { shallow } from 'enzyme';
import AdminUsersPage from './AdminUsersPage';

it('AdminUsersPage renders correct snapshot', () => {
  expect(shallow(<AdminUsersPage/>)).toMatchSnapshot();
});
