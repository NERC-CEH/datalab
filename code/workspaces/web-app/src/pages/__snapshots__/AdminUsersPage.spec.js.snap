// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminUsersPage renders correct snapshot 1`] = `
<div>
  <div
    class="Page-pageTemplate-1"
  >
    <h4
      class="MuiTypography-root MuiTypography-h4 Page-title-3 css-1yqx3hy-MuiTypography-root"
    >
      Users
    </h4>
    <div
      class="Page-contentArea-2"
    >
      <div>
        AdminUsersContainer mock
      </div>
    </div>
    <div>
      Footer mock
    </div>
  </div>
</div>
`;
